server/app/views/ProgramImageUtils.java

```diff
@@ -51,7 +51,7 @@ public Optional<ImgTag> createProgramImage(
             .withClasses(styleClasses));
   }
 
-  private static String getProgramImageAltText(ProgramDefinition program, Locale preferredLocale) {
+  public static String getProgramImageAltText(ProgramDefinition program, Locale preferredLocale) {
```

Comment:
(non-blocking) Can you check if this already has a test? Since it's being made public we should add one if it does not.